src/Illuminate/Database/Connectors/MySqlConnector.php

```diff
@@ -171,10 +171,16 @@ protected function setCustomModes(PDO $connection, array $config)
     /**
      * Get the query to enable strict mode.
      *
+     * @param  \PDO  $connection
+     *
      * @return string
      */
-    protected function strictMode()
+    protected function strictMode(PDO $connection)
     {
+        if ($connection->getAttribute(PDO::ATTR_SERVER_VERSION) >= '8.0.11') {
```

Comment:
`>= '8.0.11'` doesn't do what you want. You need `version_compare`.

Reply:
Pushed a new commit using it. Thanks!